doc/sphinx-guides/source/admin/integrations.rst

```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
 
 Researchers can use a Google Sheets add-on to search for Dataverse installation's CSV data and then import that data into a sheet. See `Avgidea Data Search <https://www.avgidea.io/avgidea-data-platform.html>`_ for details.
 
+JupyterHub
+++++++++++
+
+The Dataverse-to-JupyterHub Data Transfer Connector streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research.
+This connector facilitates seamless two-way transfer of datasets and files, emphasizing the potential of an integrated research environment.
+It is a lightweight client-side web application built using React and relying on the Dataverse External Tool feature, allowing for easy deployment on modern integration systems. Currently, it supports small to medium-sized files, with plans to enable support for large files and signed Dataverse endpoints in the future.
```

Comment:
I found the external tool manifest: https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector/-/blob/master/externalTools.json

We should also edit https://dataverse-guide--10153.org.readthedocs.build/en/10153/admin/external-tools.html#inventory-of-external-tools to add the tool there.

Reply:
Description was added to "https://dataverse-guide--10153.org.readthedocs.build/en/10153/admin/external-tools.html#inventory-of-external-tools"